### PR TITLE
fix(world-map): fix kea crash + unit display

### LINKED ISSUE #11896: Dashboard throwing error, won't load for multiple users
## Bug description

For some reason [this dashboard](https://app.posthog.com/dashboard/49928) (and only that dashboard) is throwing an error and won't load.

Screenshot 2022-09-20 at 15 17 10

## How to reproduce

1. Click link
2. Make sad face

## Environment

- [x] PostHog Cloud
- [ ] self-hosted PostHog, version/commit: please provide

## Additional context

Duplicating the dashboard doesn't fix the problem. The duplicate breaks as well.

### PR DESCRIPTION
**Pull request by mariusandra**
## Problem

Fixes https://github.com/PostHog/posthog/issues/11896

## Changes

This is a similar fix to another fix @benjackwhite had to do today 🤔 . I wonder if there are more left...

~~Also, removes the unit dropdown from this view:~~

Screenshot 2022-09-20 at 16 53 15

## How did you test this code?

Locally, no crash.

### CODE REVIEW

**Comment by Twixes** [comment]
Why hide the unit? It can still be useful for tooltips.

**Review by benjackwhite — APPROVED** [review]
Still weird but yeah 🤔

File: frontend/src/scenes/insights/views/WorldMap/worldMapLogic.tsx [hunk 1]
```diff
@@ -8,9 +8,9 @@ export const worldMapLogic = kea<worldMapLogicType>({
     props: {} as InsightLogicProps,
     key: keyForInsightLogicProps('new'),
     path: (key) => ['scenes', 'insights', 'WorldMap', 'worldMapLogic', key],
-    connect: {
+    connect: () => ({
         values: [insightLogic, ['insight', 'filters']],
```
Comment (pauldambra) [inline frontend/src/scenes/insights/views/WorldMap/worldMapLogic.tsx:12]: Should it be passing props to choose a specific `insightLogic` here? Is it assuming it is inside a `<BindLogic/>`

Comment (mariusandra) [reply frontend/src/scenes/insights/views/WorldMap/worldMapLogic.tsx:12]: Oh, there's a slightly unexpected/magical behaviour at play here: `props` are passed along if you don't specify them explicitly during `connect`. 🙃 🙈

**Pull request merged**